problems/arrayMethods.js

```diff
@@ -55,24 +86,38 @@ const onlyEvens = () => {};
  * @returns {number[]} Only number items should remain.
  */
 
-const numbersOnly = () => {};
-
+// const numbersOnly = (items) => {
+//   return items.filter((el) => {
+//     return (typeof el === "number")
+//   })
+// };
+const numbersOnly = (items) => items.filter((el) => typeof el === "number");
 /**
  * Takes in an array and checks if all elements in the array are the same.
  * Must use every
  * @param {Array} items
  * @returns {boolean} Whether or not all elements are the same.
  */
 
-const isAllSame = () => {};
+const isAllSame = (items) => {
+	return items.every((el) => {
+		for (let i = 0; i < items.length; i++) {
```

Comment:
We don't want to iterate inside of an every. Instead compare el to items[0] so that every element is the same.